packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec

```diff
@@ -8,7 +8,7 @@
 %global foreman_min_version 2.4
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.1.0
+Version: 1.0.0.rc.1
```

Comment:
This is bigger (if you ask RPM) than 1.0.0, so we won't be able to have an 1.0.0 without an epoch (🤮) and would need a 1.0.1 ir similar